package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifi/types",
-  "version": "17.1.0",
+  "version": "16.10.0-beta.1",
```

Comment:
Version should be updated, probably using `standard version`?

Reply:
It's just an old beta version. We would use beta.1, beta.2... until the BE merges and QA passes, then follow the latest version and release the official one